rebasehelper/options.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+from rebasehelper.build_helper import Builder, SRPMBuilder
+from rebasehelper.checker import checkers_runner
+from rebasehelper.output_tool import BaseOutputTool
+from rebasehelper.versioneer import versioneers_runner
+
+OPTIONS = [
```

Comment:
It would be probably better to indent a bit more to improve readability, e.g. like this:
```python
OPTIONS = [
    {
        "name": ("--version",),
        "default": False,
        "switch": True,
        "help": "show rebase-helper version and exit"
    },
    {
        "name": ("-v", "--verbose"),
        "default": False,
        "switch": True,
        "help": "be more verbose (recommended)"
    },
    [
        {
            "name": ("-b", "--build-only"),
            "default": False,
            "switch": True,
            "help": "only build SRPM and RPMs"
        },
        ...
    ],
    ...
]
```